silhouette/app/com/mohiva/play/silhouette/api/services/AuthenticatorService.scala

```diff
@@ -128,7 +150,7 @@ trait AuthenticatorService[T <: Authenticator] {
    * @param authenticator The authenticator to touch.
    * @return The touched authenticator on the left or the untouched authenticator on the right.
    */
-  protected[silhouette] def touch(authenticator: T): Either[T, T]
+  def touch(authenticator: T): Either[T, T]
```

Comment:
Is necessary to return a `Either`? i think could be use a `Option`, if is touch it returns `Some` if not `None`.
That avoid misconception